flink-formats/flink-avro/src/test/resources/META-INF/services/org.junit.jupiter.api.extension.Extension

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+org.apache.flink.util.TestLoggerExtension
```

Comment:
This is problematic if the module produces a test-jar that is used by other modules, as is the case for flink-avro. You could exclude it from the test-jar via the maven-jar-plugin.

Reply:
Added to [pom.xml](https://github.com/apache/flink/pull/18871/files#diff-2186122a79233c7232f22e689e4c3295e306f29f25080b367b24da0597ba9df1R265-R269).  Good catch!